front/pages/api/w/[wId]/apps/[aId]/state.ts

```diff
@@ -3,7 +3,7 @@ import { Op } from "sequelize";
 
 import { Authenticator, getSession } from "@app/lib/auth";
 import { App } from "@app/lib/models";
-import { withLogging } from "@app/logger/withlogging";
+import { apiError, withLogging } from "@app/logger/withlogging";
 import { AppType } from "@app/types/app";
 
 export type PostStateResponseBody = {
```

Comment:
@fontanierh nit for a small follow-up PR: add `ReturnedAPIErrorType` here. Due to the `apiError` call, it does not complain, maybe there's something to do there as well?

Reply:
Good point !